Transliteration.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Text;
+
+namespace ConsoleApp3
+{
+    class Transliteration
+    {
+        static void Main(string[] args)
+
+        {
+            try
+            {
+                Transliteration EntryPointObject = new Transliteration();
+                string inputString = args.ArrayToString();  
+                TransliterationDeciders entryTransliteration = new TransliterationDeciders();
+                entryTransliteration.AutomaticTransliteration(ref inputString);

```

Comment:
Bad idea to return result as converted string as ref argument, do it via return value